app/views/users/registrations/edit.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="text-center mt-4">
+  <h1>Edit <%= resource_name.to_s.humanize %></h1>
+</div>
+
+<div class="container form-group d-flex justify-content-center mt-4">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+
```

Comment:
Please add the missing fields for the Name, Contact Number (just copy your code from the sign up page)